demo/cli.php

```diff
@@ -0,0 +1,11 @@
+<?php declare(strict_types=1);
+
+require __DIR__.'/../vendor/autoload.php';
+
+ini_set('display_errors', '1');
```

Comment:
```suggestion
ini_set('display_errors', 'stderr');
```

Reply:
Could you explain this suggestion?

Reply:
'stderr' outputs the errors in the console, whereas the '1' would output to file.

Reply:
Right but why would it make sense here, in the context of AWS Lambda?